module/input/SensorFilter/data/config/webots/SensorFilter.yaml

```diff
@@ -80,4 +80,4 @@ initial_rpy: [0, 0.3839, 0]
 # Specify which filtering method to use, either UKF, KF, MAHONY or GROUND_TRUTH
 filtering_method: MAHONY
 
-urdf_path: "/home/nubots/NUbots/module/actuation/Kinematics/data/nugus.urdf"
+urdf_path: "/home/nubots/NUbots/shared/utility/platform/models/nugus.urdf"
```

Comment:
All these absolute paths are very suspect

Reply:
it's the path in docker